layers/+distributions/spacemacs-base/packages.el

```diff
@@ -60,7 +61,8 @@
     (progn
       (spacemacs/set-leader-keys
         "bD" 'spacemacs/ace-kill-this-buffer
-        "wC" 'spacemacs/ace-center-window
+        ;; FIXME: Needs new binding.
+        ;; "wC" 'spacemacs/ace-center-window
```

Comment:
@robbyoconnor Centering is `SPC` `w` `c` (like it was before)
 So `SPC` `w` `C` is semantically better for the destructive centering(killing other windows)